httomo/utils.py

```diff
@@ -84,7 +84,9 @@ def log_exception(output: str) -> None:
 
 
 def _parse_preview(
-    preview: List[Dict[str, int]], data_shape: Tuple[int], data_indices: List[int]
+    preview: List[Optional[Dict[str, int]]],
+    data_shape: Tuple[int],
+    data_indices: List[int],
```

Comment:
This is a nitpick, but are these changes needed for the PR to do what it's intended to do?

If I remember correctly, the `_parse_preview()` function in `utils.py` is only used by the old loader and not the new one, so I would have thought modifications to this function don't do anything of use?

This looks to be a type-hint improvement which is good, but for a function that is no longer used and unrelated to the rest of the PR.

It of course doesn't hurt to have this change, but I'd like to know if this is needed or not, can you check please?